docs/users/modules/more.md

```diff
@@ -0,0 +1,40 @@
+# More Modules
+
+Some more Modules, which we did not put into a an own category yet,
+some are also just placeholders waiting to be filled with module suggestions.
+
+## Cron Scheduling
+
+- https://github.com/AOEpeople/Aoe_Scheduler
+
+## Gift Cards
+
+- https://unirgy.com/products/ugiftcert (commercial)
```

Comment:
Unirgy no longer offers this module for M1. Probably not worth including in this documentation.

Reply:
are you sure? 
we have it, and they released a new version in april this year - they still develop it.